heka/map.jinja

```diff
@@ -65,6 +67,9 @@ RedHat:
 
 {% set remote_collector = salt['grains.filter_by']({
   'default': {
+    'amqp_port': default_amqp_port,
```

Comment:
Well, I disapprove. this is not the way the things should be structured, output backends cannot be mixed to one list like this. I need the separated approach to sustain flexibility. I can help with this/explain to details.